java/runtime/src/main/resources/ray.default.conf

```diff
@@ -31,11 +31,20 @@ ray {
     num-java-workers-per-process: 1
     /// The jvm options for java workers of the job.
     jvm-options: []
-    // Environment variables to be set on worker processes.
-    worker-env {
-      // key1 : "value1"
-      // key2 : "value2"
+
+    runtime-env: {
+         // Environment variables to be set on worker processes in current job.
+         "env-vars": {
+           // key1: "value11"
+           // key2: "value22"
+         }
+
+         "jars":[
+             // Job agent will download these jars, add them to the classpath.
+             "https://host/a.jar"
```

Comment:
Is this conf file visible for users? If it is, can we remove this `jars` from current PR? If not, can we add a comment here to notice that this `jars` API is not implemented yet?

Reply:
I'm going to remove it.